src/Raven.Server/Documents/Operations/GetReplicationActiveConnectionsInfoOperation.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Server.Documents.Commands.Replication;
+using Raven.Server.Documents.Handlers.Processors.Replication;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Operations
+{
+    internal sealed class GetReplicationActiveConnectionsInfoOperation : IMaintenanceOperation<ReplicationActiveConnectionsPreview>
```

Comment:
why we want this to be on server ? is it used somewhere beside tests?

Reply:
done